lib/screens_web/controllers/v2/screen_api_controller.ex

```diff
@@ -16,10 +16,11 @@ defmodule ScreensWeb.V2.ScreenApiController do
     end
   end
 
-  def show(conn, %{"id" => screen_id, "last_refresh" => last_refresh}) do
+  def show(%{params: params} = conn, %{"id" => screen_id, "last_refresh" => last_refresh}) do
```

Comment:
The second argument to this (and any other controller action function) is actually the same as `conn.params`. Would recommend binding `params` there instead, just to follow conventions outlined in [the guide](https://hexdocs.pm/phoenix/controllers.html#actions).
```suggestion
  def show(conn, %{"id" => screen_id, "last_refresh" => last_refresh} = params) do
```

Reply:
Thanks! That is very good to know!